### ISSUE TITLE
The ability for Vip members or higher to join teams even if they are full.

### ISSUE THREAD

**Issue by Wesdial** [Warzone/TGM#783]
**Describe the feature you'd like**
Allow Vip members or higher to join full teams. 
I suggested that a while back in the discord and it got accepted.

**Usefulness of the feature**
just another cosmetic for becoming a Vip and can also Encourage players to buy it

**Additional context**
![Vip or higher](https://user-images.githubusercontent.com/82771116/122975027-36ebac00-d393-11eb-8b93-3d7ab9743457.png)

**Comment by TheRealPear** [comment]
Wouldn't this encourage people to teamstack?

**Comment by Wesdial** [comment]
> Wouldn't this encourage people to teamstack?

no. teams are rarely full , and there ain't much maps that has low max players expect SUPERCUBE and a few maps that aren't even on the rotation,  so I don't think it will encourage team stacking that much because it will rarely happen
and also Distraction approved it, so it should happen.
